### ISSUE TITLE
Keyboard navigation through TAB doesn't work anymore in dialog boxes

### ISSUE THREAD

**Issue by sqkrv** [overtake/TelegramSwift#1027]
Since dialog boxes (e.g. message delete confirmation dialog box) have been changed from the macOS-native to custom ones, macOS keyboard navigation doesn't work anymore in them, i.e. can't move focus between controls using TAB key.
